constructor/winexe.py

```diff
@@ -242,70 +241,52 @@ def make_nsi(
     info['pre_install_desc'] = info.get('pre_install_desc', "")
     info['post_install_desc'] = info.get('post_install_desc', "")
 
-    replace = {
-        'NAME': name,
-        'VERSION': info['version'],
-        'COMPANY': info.get('company', 'Unknown, Inc.'),
-        'PLATFORM': info['_platform'],
-        'ARCH': '%d-bit' % arch,
-        'PY_VER': ".".join(py_version.split(".")[:2]),
-        'PYVERSION_JUSTDIGITS': ''.join(py_version.split('.')),
-        'PYVERSION': py_version,
-        'PYVERSION_MAJOR': py_version.split('.')[0],
-        'DEFAULT_PREFIX': info.get('default_prefix', join('%USERPROFILE%', name.lower())),
-        'DEFAULT_PREFIX_DOMAIN_USER': info.get('default_prefix_domain_user',
+    variables = {
+        'installer_name': name,
+        'installer_version': info['version'],
+        'company': info.get('company', 'Unknown, Inc.'),
+        'installer_platform': info['_platform'],
+        'arch': '%d-bit' % arch,
+        'py_ver': ".".join(py_version.split(".")[:2]),
+        'pyversion_justdigits': ''.join(py_version.split('.')),
+        'pyversion': py_version,
+        'pyversion_major': py_version.split('.')[0],
```

Comment:
This looks like something that can be done by jinja. That would make it more obvious in the template and developers don't have to go look for the definition of these similar python variables.

Reply:
Agreed. I didn't want to move too many things at first so we can do a 1:1 translation of variables and so on. We can then optimize the Jinja/Python split better. WDYT if we do this in a new PR?

Reply:
That sounds fair to me.